hopsworks-common/src/main/java/io/hops/hopsworks/common/serving/tf/LocalhostTfServingController.java

```diff
@@ -17,268 +17,95 @@
 package io.hops.hopsworks.common.serving.tf;
 
 import com.google.common.io.Files;
+import io.hops.hopsworks.common.dao.hdfs.inode.Inode;
+import io.hops.hopsworks.common.dao.hdfs.inode.InodeFacade;
 import io.hops.hopsworks.common.dao.project.Project;
-import io.hops.hopsworks.common.dao.serving.TfServing;
-import io.hops.hopsworks.common.dao.serving.TfServingFacade;
+import io.hops.hopsworks.common.dao.serving.Serving;
+import io.hops.hopsworks.common.dao.serving.ServingFacade;
 import io.hops.hopsworks.common.dao.user.Users;
-import io.hops.hopsworks.exceptions.KafkaException;
-import io.hops.hopsworks.exceptions.ProjectException;
-import io.hops.hopsworks.restutils.RESTCodes;
-import io.hops.hopsworks.exceptions.ServiceException;
-import io.hops.hopsworks.exceptions.UserException;
 import io.hops.hopsworks.common.security.CertificateMaterializer;
-import io.hops.hopsworks.common.serving.KafkaServingHelper;
+import io.hops.hopsworks.common.serving.ServingException;
+import io.hops.hopsworks.common.serving.ServingWrapper;
 import io.hops.hopsworks.common.util.OSProcessExecutor;
 import io.hops.hopsworks.common.util.ProcessDescriptor;
 import io.hops.hopsworks.common.util.ProcessResult;
 import io.hops.hopsworks.common.util.Settings;
+import io.hops.hopsworks.restutils.RESTCodes;
 
 import javax.ejb.EJB;
 import javax.ejb.Stateless;
 import javax.ejb.TransactionAttribute;
 import javax.ejb.TransactionAttributeType;
 import javax.enterprise.inject.Alternative;
+import java.io.FileNotFoundException;
 import java.io.IOException;
 import java.nio.charset.Charset;
 import java.nio.file.Path;
 import java.nio.file.Paths;
-import java.util.ArrayList;
-import java.util.Date;
 import java.util.List;
-import java.util.UUID;
 import java.util.concurrent.ThreadLocalRandom;
 import java.util.concurrent.TimeUnit;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
 import static io.hops.hopsworks.common.hdfs.HdfsUsersController.USER_NAME_DELIMITER;
+import static io.hops.hopsworks.common.serving.LocalhostServingController.PID_STOPPED;
+import static io.hops.hopsworks.common.serving.LocalhostServingController.SERVING_DIRS;
 
+/**
+ * Localhost Tensorflow Serving Controller
+ *
+ * Launches/Kills a local tensorflow-serving-server for serving tensorflow Models
+ */
 @Alternative
```

Comment:
This is not an `alternative` anymore. `alternative` is used when there are multiple classes implementing the same interface and glassfish needs to know which one to use.
This is not an interface anymore

Reply:
Done